installer/x86_64/install.sh

```diff
@@ -452,7 +452,13 @@ unzip -op $ONIE_INSTALLER_PAYLOAD "$FILESYSTEM_DOCKERFS" | tar xz $TAR_EXTRA_OPT
 
 if [ "$install_env" = "onie" ]; then
     # Store machine description in target file system
-    cp /etc/machine.conf $demo_mnt
+    if [ -f /etc/machine-build.conf ]; then
+        # onie_ variable are generate at runtime.
+        # they are no longer hardcoded in /etc/machine.conf
+        set | grep onie_ > $demo_mnt/machine.conf
```

Comment:
Why use 'set' instead of 'env'?
Suggest grep 'onie_' at the line beginning. Maybe
```
env | grep ^onie_
```
Not sure it is available in ONIE environment.

Reply:
since the variables are not exported, so they are not env